ogr/abstract.py

```
@@ -267,6 +267,16 @@ def change_token(self, new_token: str):
         """
         raise NotImplementedError
 
+    def get_file_content(self, path: str, revision="master") -> Optional[bytes]:
+        """
+        Get a content of the file in the repo.
+
+        :param revision: branch or commit (defaults to master)
+        :param path: str
+        :return: bytes or None if there is no such file
```

please raise an Exception if the file does not exist

---

and returning str will be easier for everyone